lab_1/main.py

```diff
@@ -12,7 +13,11 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    if isinstance(text, str) and text:
```

Comment:
better write `if not isinstance(text, str) and text: return []` here. And then implement main logic without any nesting. Nesting is bad, especially when it can be avoided. 

*styling
1. avoid nesting
2. separate params checks from main logic